style.css

```diff
@@ -6,22 +6,23 @@ header {
   display: flex;
   flex-direction: row;
   justify-content: center;
+  background-color: rgba(0, 0, 0, 0);
 }
 
 .head1 {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding-top: 25%;
+  padding-top: 10%;
   gap: 58px;
   position: relative;
-  height: 726px;
+  height: 100vh;
   top: 10%;
   background-image: url('./Assets/background.png');
-  background-position-x: center;
-  background-position-y: center;
-  background-size: contain;
-  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
   width: 100%;
 
   /* max-width: fit-content; */
```

Comment:
[OPTIONAL] Kindly delete all uncommented codes on your codebase. It will keep your codebase clean and organized. 📦📦